### PR TITLE
feat: add genese solution

### PR DESCRIPTION
**Pull request by aayush105**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

### CODE REVIEW

File: data/companies.json [hunk 1]
```diff
@@ -336,6 +336,22 @@
         "youtube": "https://www.youtube.com/c/Fusemachines"
       }
     },
+    {
+      "name": "Genese Solution",
+      "establishedAt": 1555006500,
```
Comment (thenishantsapkota) [inline data/companies.json:341]: What's the source for this?

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Review by drishyakarki — CHANGES_REQUESTED** [review]
Please make the necessary changes made by @thenishantsapkota

File: data/companies.json (hunk 1 above)
Comment (aayush105) [reply data/companies.json:341]: > What's the source for this?
Here it is :
Screenshot from 2023-10-21 10-28-18

**Review by thenishantsapkota — APPROVED** [review]
LGTM!
Thank you for your contribution @aayush105

**Review by drishyakarki — APPROVED** [review]

**Comment by aayush105** [comment]
@thenishantsapkota  why it is marked as invalid label? which information seems wrong to you? Can provide the review?

**Comment by thenishantsapkota** [comment]
Same for this the format for phone number was wrongly merged.

**Pull request merged**